chingu-coders/Voyage2-Turtles-02
# rotating backgrounds

## Purpose
* Having changing backgrounds to offer different visuals

## Approach/Fixes
* Each time the tab is opened/refreshed a new background image is used
* Background images are selected from a pool of saved images
* Images are categorized into daytime/nighttime
* A function takes the user's local time to determine which category of images to select from
* Credit to the photographer/contributor is given at the bottom with a link to their work

## Testing
* [Check out this pull request](https://help.github.com/articles/checking-out-pull-requests-locally/)
* Refresh or open the page a few times to make sure the background image changes
* Check to see that the photographer/contributor changes
* Check to see that the associated link matches the background image or photographer/contributor's work
* Check during both daytime/nighttime hours to see difference in images
